automation/check-patch.e2e-k8s.sh

```diff
@@ -27,7 +27,7 @@ main() {
     make cluster-up
     trap teardown EXIT SIGINT SIGTERM SIGSTOP
     make cluster-sync
-    make E2E_TEST_ARGS="-ginkgo.noColor" test/e2e
+    make E2E_TEST_TIMEOUT=1h E2E_TEST_ARGS="-ginkgo.noColor " test/e2e
```

Comment:
Do we need the space after `noColor`? Was there a bug?

Reply:
I have just copy paste it from ocp lane, there is no bug, do I change it ?